pyannote/audio/tasks/segmentation/voice_activity_detection.py

```diff
@@ -74,6 +74,8 @@ class VoiceActivityDetection(SegmentationTaskMixin, Task):
     metric : optional
         Validation metric(s). Can be anything supported by torchmetrics.MetricCollection.
         Defaults to AUROC (area under the ROC curve).
+    cache : str, optional
+        path to file where to write or load task caches
```

Comment:
Can you use the same docstring as the one from `SegmentationTask`? 
And this for all sub-segmentation tasks?